packages/mobile/src/screens/explore-screen/SearchExploreScreen.tsx

```diff
@@ -0,0 +1,180 @@
+
+  // Derived data
+  const filteredTiles = tiles.filter((tile) => {
+    const isPremiumTracksTile =
+      tile.variant === ExploreCollectionsVariant.DIRECT_LINK &&
+      tile.title === PREMIUM_TRACKS.title
+    return !isPremiumTracksTile || isUSDCPurchasesEnabled
+  })
+
+  return (
+    <Screen url='Feed' header={() => <></>}>
+      <ScreenContent>
+        <Flex>
+          <ImageBackground source={imageSearchHeaderBackground}>
+            <Flex pt='unit14' ph='l' pb='l' gap='l'>
+              <Flex direction='row' gap='m'>
+                <AccountPictureHeader onPress={handleOpenLeftNavDrawer} />
+                <Text variant='heading' color='staticWhite'>
+                  {messages.explore}
+                </Text>
+              </Flex>
+              <Text variant='title' color='staticWhite'>
+                {messages.description}
+              </Text>
+              <Flex>
+                <TextInput
+                  label='Search'
+                  // ref={searchBarRef}
+                  placeholder={messages.searchPlaceholder}
+                  // onChange={handleSearch}
+                  // onClear={handleClearSearch}
+                  size={TextInputSize.SMALL}
+                  startIcon={IconSearch}
+                />
+              </Flex>
+            </Flex>
+          </ImageBackground>
+        </Flex>
+        <SearchCategoriesAndFilters />
+
+        <ScrollView>
+          <Flex direction='column' ph='l' pt='xl' pb='3xl' gap='2xl'>
+            <ExploreCarousel
+              title={messages.featuredPlaylists}
+              list={
+                <CollectionList
+                  collectionIds={exploreContent?.featuredPlaylists || []}
+                />
+              }
+            />
+
+            <ExploreCarousel
+              title={messages.artistSpotlight}
+              list={<UserList profiles={featuredArtists} />}
+            />
+
+            <ExploreCarousel
+              title={messages.labelSpotlight}
+              list={<UserList profiles={featuredLabels} />}
+            />
+
+            <Flex justifyContent='center' gap='l'>
+              <Text variant='title' size='l' textAlign='center'>
+                Explore By Mood
+              </Text>
+              <Flex wrap='wrap' direction='row' justifyContent='center' gap='s'>
+                {Object.entries(MOODS)
+                  .sort()
+                  .map(([mood, moodInfo]) => (
+                    <Paper
+                      direction='row'
+                      key={moodInfo.label}
+                      pv='l'
+                      ph='xl'
+                      gap='m'
+                      borderRadius='m'
+                      border='default'
+                      backgroundColor='white'
+                    >
+                      <Image
+                        source={moodMap[moodInfo.label]}
+                        style={styles.emoji}
```

Comment:
so here it could be style={{height: spacing.unit5, width: spacing.unit5}} or something